experimenter/experimenter/nimbus-ui/src/components/PageHome/filterExperiments.ts

```diff
@@ -82,6 +83,12 @@ const experimentFilters: { [key in FilterValueKeys]: ExperimentFilter<key> } = {
     }
     return targetingConfigMatch;
   },
+  takeaways: (option, experiment) => {
+    return (
+      (experiment.takeawaysQbrLearning && option.value === "QBR_LEARNING") ||
+      (experiment.takeawaysMetricGain && option.value === "METRIC_GAIN")
+    );
+  },
```

Comment:
🎉

Reply:
Oh we should make these into proper constants at some point.  There's probably a way to pass the enum into gql and have it dump out a ts enum.